static/app/views/performance/transactionDetails/eventMetas.tsx

```diff
@@ -236,4 +237,32 @@ function HttpStatus({event}: {event: Event}) {
   return <React.Fragment>HTTP {tag.value}</React.Fragment>;
 }
 
+/*
+  TODO: Ash
+  I put this in place as a temporary patch to prevent successful frontend transactions from being set as 'unknown', which is what Relay sets by default
+  if there is no status set by the SDK. In the future, the possible statuses will be revised and frontend transactions should properly have a status set.
+  When that change is implemented, this function can simply be replaced with:
+
+  event.contexts?.trace?.status ?? '\u2014';
+*/
+
+function getStatusBodyText(
+  project: AvatarProject | undefined,
+  event: EventTransaction,
+  meta: TraceMeta | null
+): string {
+  const isFrontendProject = frontend.some(val => val === project?.platform);
+
+  if (
+    isFrontendProject &&
+    meta &&
+    meta.errors === 0 &&
```

Comment:
is errors a number?

Reply:
Yep 👍

Reply:
https://github.com/getsentry/sentry/blob/ff08c0c06d39924938809ddc2d4e591a599d2880/static/app/utils/performance/quickTrace/types.tsx#L107